### PR TITLE
Revert lg profile friendly name

### PR DESCRIPTION
**Pull request by disgustipated**
Add . back in to friendly name for LG TVs, caused rexeg to match unexpected friendly names of other devices Fixes https://github.com/jellyfin/jellyfin-plugin-dlna/issues/76

### CODE REVIEW

**Comment by Shadowghost** [comment]
This PR has no changes

**Comment by disgustipated** [comment]
Missed the commit, should be there now

**Comment by disgustipated** [comment]
Good point, I'll see if i can test that

**Comment by FlashSystems** [comment]
> I wonder if `Identification` conditions are `OR`ed together because otherwise it should only apply if the header also include `LG`, no? If yes, maybe they should be `AND`

I've taken a look at the code. The two identification parts are used at different locations within the code that seem to be completely unrelated. The PlayToManager calls GetProfile only passing the device identification (no headers here): https://github.com/jellyfin/jellyfin-plugin-dlna/blob/f7a4beaa5e12cfa40c38df49b97a23556a285773/src/Jellyfin.Plugin.Dlna/PlayTo/PlayToManager.cs#L211

The DlnaManager only uses the header fields and no device identification:
https://github.com/jellyfin/jellyfin-plugin-dlna/blob/f7a4beaa5e12cfa40c38df49b97a23556a285773/src/Jellyfin.Plugin.Dlna/DlnaManager.cs#L453

What makes everything really confusing that both methods are called `GetProfile` but do completely different things depending on if you pass a class that implements `IHeaderDictionary` or `DeviceIdentification`.

At first glance there is neither an AND nor an OR. They seem to be used for different things...

**Comment by andekande** [comment]
You are right, there is no Regex needed. The simple contains check suffice.
However as its touched anyways one can consider to even add Manufacturer to make matches not to verbose:
    <FriendlyName>LG</FriendlyName>
    <Manufacturer>LG</Manufacturer>

Matches this strings of my TV:
FriendlyName: [LG] webOS TV OLED123456
Manufacturer: LG Electronics

File: src/Jellyfin.Plugin.Dlna/Profiles/Xml/LG Smart TV.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   xmlns:xsd="http://www.w3.org/2001/XMLSchema">
   <Name>LG Smart TV</Name>
   <Identification>
-    <FriendlyName>LG*</FriendlyName>
+    <FriendlyName>LG.*</FriendlyName>
```
Comment (Bond-009) [inline src/Jellyfin.Plugin.Dlna/Profiles/Xml/LG Smart TV.xml:6]:
```suggestion
    <FriendlyName>LG</FriendlyName>
```

**Review by Bond-009 — CHANGES_REQUESTED** [review]

**Review by Bond-009 — APPROVED** [review]

**Pull request merged**